source/ed/connectors/gtfs_parser.cpp

```diff
@@ -989,7 +1002,11 @@ void StopTimeGtfsHandler::finish(Data& data) {
 }
 
 static int to_utc(const std::string& local_time, int utc_offset) {
-    return time_to_int(local_time) - utc_offset;
+    int local = time_to_int(local_time);
```

Comment:
switched it to -infinite because actual times can be negative after utc offset, and also for clarity of debug